htdocs/compta/ajax/addpayment.php

```diff
@@ -0,0 +1,277 @@
+<?php
+/* Copyright (C) 2022-2023  Easya Solutions         <support@easya.solutions>
+ * Copyright (C) 2023       Sylvain Legrand         <technique@infras.fr>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/**
+ *       \file       htdocs/compta/ajax/addpayment.php
+ *       \brief      File to return Ajax response on payment process
+ */
+
+//if (! defined('NOREQUIRESOC'))   define('NOREQUIRESOC', '1');
+//if (! defined('NOCSRFCHECK'))    define('NOCSRFCHECK', '1');
+if (! defined('NOTOKENRENEWAL')) define('NOTOKENRENEWAL', '1');
+if (! defined('NOREQUIREMENU'))  define('NOREQUIREMENU', '1'); // If there is no menu to show
+if (! defined('NOREQUIREHTML'))  define('NOREQUIREHTML', '1'); // If we don't need to load the html.form.class.php
+
+require '../../main.inc.php';
+require_once DOL_DOCUMENT_ROOT.'/compta/paiement/class/paiement.class.php';
+require_once DOL_DOCUMENT_ROOT.'/compta/facture/class/facture.class.php';
+require_once DOL_DOCUMENT_ROOT.'/compta/bank/class/account.class.php';
+require_once DOL_DOCUMENT_ROOT.'/societe/class/societe.class.php';
+
+$langs->loadLangs(array('banks', 'bills', 'companies', 'multicurrency'));
+$error = 0;
+$invoice_id = 0;
+$errors_msg = array();
+
+/*
+ * View
+ */
+top_httphead();
```

Comment:
The page return a json file so the top_httphead must be
top_httphead('application/json')

Reply:
Thanks @eldy for your return

Corrected !